src/Sylius/Bundle/CoreBundle/Cart/ItemResolver.php

```diff
@@ -60,22 +60,22 @@ class ItemResolver implements ItemResolverInterface
     protected $channelContext;
 
     /**
-     * @param CartProviderInterface          $cartProvider
+     * @param CartContextInterface          $cartContext
```

Comment:
One of the last places, where we still have parameters aligned vertically... Good, old times. Can we remove this alignment?